hw/top_earlgrey/sw/autogen/top_earlgrey.h

```diff
@@ -1617,11 +1617,11 @@ typedef enum top_earlgrey_pinmux_outsel {
  * Power Manager Wakeup Signals
  */
 typedef enum top_earlgrey_power_manager_wake_ups {
-  kTopEarlgreyPowerManagerWakeUpsSysrstCtrlAonAonSysrstCtrlWkupReq = 0, /**<  */
-  kTopEarlgreyPowerManagerWakeUpsAdcCtrlAonDebugCableWakeup = 1, /**<  */
-  kTopEarlgreyPowerManagerWakeUpsPinmuxAonAonWkupReq = 2, /**<  */
+  kTopEarlgreyPowerManagerWakeUpsSysrstCtrlAonWkupReq = 0, /**<  */
```

Comment:
Gee, how many different variables for the same thing...

Reply:
yeah..this is why @timothytrippel and i had a chat on what best to do. 
because this really is the same thing..except we can't use the above info in `dif_pwrmgr.c`

Reply:
Yes, this has to do with the lack of global files in sw/device. I imagine we may end up with a better solution at some point. Okay for now...

Reply:
do you have a look at the other issue i referenced above and leave your opinion?